maintenance/passport-azure-ad/lib/pem.js

```diff
@@ -44,31 +25,33 @@
  */
 
 /* eslint-disable max-len */
-const CERT_REGEX = /-----BEGIN\sCERTIFICATE-----\s+([a-zA-Z0-9+\/=\r\n]+)\s+-----END\sCERTIFICATE-----/;
-const PRIVATE_KEY_REGEX = /-----BEGIN\sPRIVATE\sKEY-----\s+([a-zA-Z0-9+\/=\r\n]+)\s+-----END\sPRIVATE\sKEY-----/;
+const CERT_REGEX = /-----BEGIN\sCERTIFICATE-----\s+([a-zA-Z0-9+/=\r\n]+)\s+-----END\sCERTIFICATE-----/;
```

Comment:
New regex is slightly different, is this correct?

Reply:
Hmmm the linter was throwing a "no-useless-escape" error, but it looks like `/` should still be escaped, so I've added a rule disable for these few lines after checking the regex.